sdk/analytics/src/main/java/com/klaviyo/analytics/model/Event.kt

```diff
@@ -29,13 +29,17 @@ class Event(val metric: EventMetric, properties: Map<EventKey, Serializable>?) :
                 null
             }
         }
+
+        @JvmSynthetic
```

Comment:
Previously, the fluent setter and property setter conflicted. This hides the auto-generated setter, and keeps the fluent setter, since they do the same thing anyway.

Reply:
"fluent setter" refers to method chaining, e.g. `.setValue(123).setUniqueId("ID")`

The property's setter conflicts with the version above that returns `this`, so i just keep the one that returns `this` and hid the one that returns void.